articles/aws-api-setup.md

```diff
@@ -19,7 +19,7 @@ This is a one time setup
   ![](/media/articles/aws-api-setup/aws-api-setup-2.png)
 
 <% if (account.userName) { %>
-3. Download the metadata from <a href="https://@@account.namespace@@/samlp/metadata/@@account.clientId@@" target="_new">https://@@account.namespace@@/samlp/metadata/@@account.clientId@@</a> and upload it here.
+3. Download the metadata from <a href="https://@@account.namespace@@/samlp/metadata/@@account.clientId@@">https://@@account.namespace@@/samlp/metadata/@@account.clientId@@</a> and upload it here.
```

Comment:
Your preference is that they open in another tab, but that isn't universally agreed upon. The nice thing about NOT putting target=blank is that you can use CTRL+Click and get a new window and other people who don't like this can use regular click. I am strongly in favor of letting the user choose what happens in their browser.

You can't trick people to stay on a certain page. We aren't going lose people because we don't open in a new window. Everyone knows how the back button works. If somebody is invested in a tutorial they are going to return to it if they accidentally open a link in the same window (which by the way is the default expectation of any link you open on the internet).

You also have to consider the mobile browsing experience where tabs are not nearly as easy as back. If you go to news.google.com and open any article it opens in a new tab. If you do this on your mobile device you will shortly become infuriated because you read an article and cannot use the back button. The correct UX for google news would be to let the user chose what window the article opened in. I would choose to use the same page then click back when I am ready to read more on google news.

http://www.smashingmagazine.com/2008/07/01/should-links-open-in-new-windows/